### DISCUSSION TITLE
How to keep Jupyter In[1..] and Out[..] cell numbers?

### DISCUSSION THREAD

**Discussion by Naereen** [executablebooks/meta#260] [Q&A]
Hello there,
I actually really like Jupyter cell numbers, and I can't find on the doc (of [jupyter-book](https://jupyterbook.org/content/code-outputs.html#content-code-outputs) or my [myst-nb](https://myst-nb.readthedocs.io/en/latest/use/formatting_outputs.html#use-format-cutomise)) if there is an option to display the cell numbers (starting fresh from In[1]/Out[1] for the first cell and increasing) in Jupyter notebook documents that get converted to HTML/LaTeX?
I would really appreciate it!
Thanks in advance, regards from France, @Naereen

**Comment by chrisjsewell** [comment]
Hi @Naereen, not at present I'm afraid. There is open issue which you can upvote (i.e. add a 👍 to the initial comment): https://github.com/executablebooks/MyST-NB/issues/64

**Reply by Naereen** [reply]
Hi @chrisjsewell, thanks for the reply, I have voted there.
